acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisDepositSender.java

```diff
@@ -44,6 +47,11 @@ public String sendValidatorDeposits(final BesuNode eth1Node, final int numberOfV
     container.start();
     Waiter.waitFor(() -> assertThat(container.isRunning()).isFalse());
     container.stop();
-    return validatorKeys.toString();
+    // because we have introduced some logging, only pick lines which start with a - {
+    return validatorKeys
+        .toString()
+        .lines()
+        .filter(s -> s.startsWith("- {"))
+        .collect(Collectors.joining(System.lineSeparator()));
```

Comment:
This isn't the right approach.  If we're outputting to STDOUT, we need to make sure logging is either disabled or sent to STDERR.  Otherwise people who run this tool in real life will get corrupt output.

Reply:
It's because of the limitation of "testcontainer" docker API we are using that captures whole of standard/error output. A more better approach, which I tested, is to mount the "keys" file with `--output-path` and read that file after running the command. However, in Windows environment this usually cause an issue because we would have "share" the host location via docker preference. In another project of ours, we used user's home folder and create a temp directory there which can be mounted to docker container, I wonder if we can use same approach. 

Last, but not least, is that I can remove the "info" logging that I introduced so that it only outputs the yaml keys on std out.

Reply:
It's not really docker related - if we're saying we'll output the content to STDOUT, we should *only* output the content to STDOUT but the logs are going there as well (they don't go to STDERR by default).  We can configure test containers to capture STDOUT and STDERR separately if need be.

I suspect just removing the log messages is probably the simplest approach though.

Reply:
Removed additional logging for "yaml keys on standard out" mode.